junebug/api.py

```diff
@@ -339,4 +343,58 @@ def create_router(self, request, body):
 
     @app.route('/health', methods=['GET'])
     def health_status(self, request):
-        return response(request, 'health ok', {})
+        if self.config.rabbitmq_management_interface:
+            rabbitmq_management_client = RabbitmqManagementClient(
+                self.config.rabbitmq_management_interface,
+                self.amqp_config['username'],
+                self.amqp_config['password'])
```

Comment:
This will be creating a new client for every request, which could mean a lot of parallel requests to the management interface if Junebug is handling a lot of parallel requests to this endpoint.

This is probably a rare case, but maybe we can create the client in `setup`, if the config value is set.